client/src/data_layer_configs/lead_service_lines_by_water_systems_config.ts

```diff
@@ -32,12 +32,12 @@ const LEGEND_COLOR_MAPPING = [
 const leadConnectionLegendInterpolation = [
   'interpolate',
   ['linear'],
-  ['get', 'lead_connections_count'],
+  ['/', ['get', 'lead_connections_count'], ['get', 'service_connections_count']],
```

Comment:
what does this do?

Reply:
Added comment: provides ratio of lead lines : all lines